tensorflow/stream_executor/stream_executor_pimpl.h

```diff
@@ -378,27 +378,15 @@ class StreamExecutor {
       std::vector<std::unique_ptr<dnn::ConvolveExecutionPlan>> *out_exec_plans);
 
   port::Status GetFusedConvolveExecutionPlans(
-      dnn::ConvolutionKind kind, dnn::DataType element_type, Stream *stream,
+      dnn::ConvolutionKind kind, dnn::DataType element_type,
+      double conv_input_scale, double side_input_scale, Stream *stream,
       const dnn::BatchDescriptor &input_descriptor,
       const dnn::FilterDescriptor &filter_descriptor,
       const dnn::BatchDescriptor &bias_descriptor,
       const dnn::BatchDescriptor &output_descriptor,
       const dnn::ConvolutionDescriptor &convolution_descriptor,
       std::vector<std::unique_ptr<dnn::ConvolveExecutionPlan>>
-          *out_exec_plans) {
-    dnn::DnnSupport *dnn_support = AsDnn();
-    if (dnn_support) {
-#if GOOGLE_CUDA
```

Comment:
Please keep all GOOGLE_CUDA references in the headers, as GOOGLE_CUDA is often defined in the user land and propagated into the included headers (but not .cc files). Internally we rely on this kind of propagation.